_guides/upgrade.md

```diff
@@ -57,3 +59,22 @@ In addition to the items listed in this guide, you may want to enable or configu
 - We have updated our build pipeline to better reflect our browser support. The Lounge now supports [browsers released less than a year ago](https://browserl.ist/?q=last+1+year%2C+firefox+esr) (and [Firefox ESR](https://www.mozilla.org/en-US/firefox/organizations/)), while we used to support the [last 2 versions of all browsers](https://browserl.ist/?q=last+2+versions). In practice, this drops support for some of the least used browsers on mobile (QQ browser, UC browser, Android Browser, Baidu Browser, Blackberry Browser, IE Mobile, Opera Mini, and Opera Mobile), and IE 10/11 on desktop. It however gives slightly more support flexibility for earlier versions of Chrome (62+), Firefox (56+ and ESR), and Opera (48+).
 
 {% include abbreviations.md %}
+
+
+# Upgrading to The Lounge v4
```

Comment:
Put v4 above v3.

And use `##` for this heading, and `###` for the lower ones, as the main page title itself is also a <h1>